lib/msf/core/exploit/remote/http_client.rb

```diff
@@ -35,7 +35,7 @@ def initialize(info = {})
     register_advanced_options(
       [
         OptString.new('UserAgent', [false, 'The User-Agent header to use for all requests',
-          Rex::Proto::Http::Client::DefaultUserAgent
+                                    Rex::UserAgent.session_agent
```

Comment:
Just a minor white space nitpick since Rubocop doesn't run on lib files.
```suggestion
          Rex::UserAgent.session_agent
```

Reply:
Done.